cmd/basecoin/commands/init.go

```diff
@@ -64,7 +67,24 @@ func initCmd(cmd *cobra.Command, args []string) error {
 		return errors.New("Address must be 20-bytes in hex")
 	}
 
-	genesis := GetGenesisJSON(viper.GetString(FlagChainID), userAddr)
+	var options []string
+	var optionsStr string
+	sep := ",\n      "
+	optionsRaw := viper.GetStringSlice(FlagOption)
+	if len(optionsRaw) > 0 {
+		optionsStr = sep
+		for i := 0; i < len(optionsRaw); i++ {
+			s := strings.SplitN(optionsRaw[i], "/", 3)
+			if len(s) != 3 {
+				return errors.New("Genesis option must be in the format <app>/<option>/<value>")
+			}
+			option := `"` + s[0] + `/` + s[1] + `", "` + s[2] + `"`
+			options = append(options, option)
+		}
+	}
+	optionsStr += strings.Join(options[:], sep)
```

Comment:
s/+=/=/ since it just defaults to the empty string above and put this into the if statement

Reply:
Moved to the if statement, replaced += ... with = sep + ...